.github/workflows/pull-request-ci.yaml

```diff
@@ -217,8 +217,8 @@ jobs:
           OWNER: ${{ github.repository_owner }}
           DEBUG_MODE: true
           GKE_NODE_COUNT: 1
-          GKE_PREEMPTIBLE: true
-          GKE_INSTANCE_TYPE: n1-highcpu-16
+          GKE_PREEMPTIBLE: false
+          GKE_INSTANCE_TYPE: n1-highcpu-32
```

Comment:
In my tests, `n2d-highcpu-16` was enough (but probably _barely_ so); would it make more sense to use `n2d-highcpu-32` instead? It's actually slightly cheaper for non-preempt, likely because of the AMD CPUs:

type | vCPUs | RAM | Price | Price (Preempt)
-- | -- | -- | -- | --
n2d-highcpu-16 | 16 | 16 GB | $0.549376 | $0.132992
n2d-highcpu-32 | 32 | 32 GB | $1.098752 | $0.265984
n1-highcpu-32 | 32 | 28.80GB | $1.2496 | $0.2629

All prices are for `europe-west4`.